devicetypes/smartthings/zooz-4-in-1-sensor.src/zooz-4-in-1-sensor.groovy

```diff
@@ -236,3 +282,71 @@ private secure(physicalgraph.zwave.Command cmd) {
 private secureSequence(commands, delay = 200) {
 	delayBetween(commands.collect{ secure(it) }, delay)
 }
+
+def getConfigurationCommands() {
+	log.debug "getConfigurationCommands"
+	def result = []
+
+	if (isEverspringSP815()) {
+		Integer temperatureAndHumidityReport = (settings.temperatureAndHumidityReport as Integer) ?: everspringDefaults[1]
+		Integer retriggerIntervalSettings = (settings.retriggerIntervalSettings as Integer) ?: everspringDefaults[2]
+
+		if (!state.temperatureAndHumidityReport) {
+			state.temperatureAndHumidityReport = getEverspringDefaults[1]
+		}
+		if (!state.retriggerIntervalSettings) {
+			state.retriggerIntervalSettings = getEverspringDefaults[2]
+		}
+
+		if (!state.configured || (temperatureAndHumidityReport != state.temperatureAndHumidityReport || retriggerIntervalSettings != state.retriggerIntervalSettings)) {
+
```

Comment:
Might be an accidental extra blank line here, but this function is MUCH easier to read now!

Reply:
yes, looks better now! already deleted an extra line